src/init.lua

```diff
@@ -239,12 +271,12 @@ function WindShake:Update(deltaTime: number)
 
 	debug.profileend()
 
-	workspace:BulkMoveTo(partList, cframeList, Enum.BulkMoveMode.FireCFrameChanged)
+	workspace:BulkMoveTo(partList :: { any }, cframeList, Enum.BulkMoveMode.FireCFrameChanged)
```

Comment:
Does this cast need to be here? Isn't `partList` already typed as `{ any }` by `WindShake._partList`?

Reply:
Yep. I did actually fix this in my fork already. Guess I forgot to commit it <_>